devlab/Vagrantfile

```diff
@@ -68,6 +75,15 @@ Vagrant.configure(2) do |config|
           thisnode.vm.provision "shell",
             path: "./provision/keys.sh",
             args: [ "--public-key", File.read("#{ENV["HOME"]}/#{options['public_key_path']}").strip() ]
+          if nodedata['installer'] == 'devstack'
+            thisnode.vm.provision "file",
+              source: "./provision/devstack/local_#{nodename}.conf", 
+              destination: "local.conf"
+            thisnode.vm.provision "shell",
+              privileged: false,
+              path: "./provision/devstack/devstack.sh",
+              args: [ "--branch", "stable/#{nodename}" ]
```

Comment:
AFAIK devstack creates neutron networks, router, floating IP, and some other stuff by default. Can we NOT do this some way?...